### ISSUE TITLE
Accessibility violation | You can't press the 'X' button with the keyboard

### ISSUE THREAD

**Issue by thomas-siozos** [Vuepic/vue-datepicker#908]
**Describe the bug**
You can't press the 'X' button with your keyboard.

**To Reproduce**
1. Go to the date picker.
2. Pick a date.
3. You can't press the 'X' button with the keyboard to clear the date.

**Expected behavior**
You should be able to press the 'X' button with the keyboard to clear the date.

**Desktop & mobile (please complete the following information):**
 - Browser
